### PR TITLE
MNT: Prepare hi-ml-multimodal for PyPI

### PR DESCRIPTION
**Pull request by fepegar**
- Simplified `setup.py` in `hi-ml-multimodal` as we are not using CI for these purposes.
- Set initial PyPI version to `0.1.0` as recommended in [Semantic Versioning 2.0.0](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase)
- Added `MANIFEST.in`, needed to include files such as `requirements_*.txt`, `README` and `LICENSE` in the package distribution
- Tried TestPyPI by running the commands below (from [Packaging Python Projects](https://packaging.python.org/en/latest/tutorials/packaging-projects/))

```console
cd hi-ml-multimodal
python3 -m pip install --upgrade build
python3 -m build
python3 -m pip install --upgrade twine
python3 -m twine upload --repository testpypi dist/*
```

### CODE REVIEW

**Comment by fepegar** [comment]
@Shruthi42, feel free to commit a more appropriate author name for the package.

**Review by Shruthi42 — APPROVED** [review]

File: hi-ml-multimodal/setup.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/microsoft/hi-ml",
-    author="Microsoft Research Cambridge Medical Imaging Team ",
+    author="Biomedical Imaging Team @ Microsoft Health Futures",
```
Comment (Shruthi42) [inline hi-ml-multimodal/setup.py:25]: Thanks for changing this!

Comment (fepegar) [reply hi-ml-multimodal/setup.py:25]: Thanks for looking into it.

File: hi-ml-multimodal/setup.py [hunk 2]
```diff
@@ -3,85 +3,36 @@
...
+        "Intended Audience :: Science/Research",
         "Topic :: Scientific/Engineering :: Medical Science Apps.",
-        'License :: OSI Approved :: MIT License',
-        'Programming Language :: Python :: 3.7'
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python :: 3.7"
     ],
-    keywords='HealthIntelligence',
-    license='MIT License',
+    keywords="HealthIntelligence",
```
Comment (Shruthi42) [inline hi-ml-multimodal/setup.py:34]: Some suggestions for more keywords we could add: Health Futures, Multimodal

Comment (fepegar) [reply hi-ml-multimodal/setup.py]: I've just removed this. Probably not very useful anyway.

**Review by ozan-oktay — APPROVED** [review]
Thanks 👍

**Pull request merged**